ts/features/itwallet/lifecycle/store/actions/index.ts

```diff
@@ -8,17 +8,36 @@ export const itwLifecycleStateUpdated = createStandardAction(
   "ITW_LIFECYCLE_STATE_UPDATED"
 )<ItwLifecycleState>();
 
+/**
+ * @internal To fully reset the Wallet Instance, dispatch `itwLifecycleWalletReset`.
+ *
+ * Action used to reset IT Wallet reducers only.
+ */
+export const itwLifecycleReducersReset = createStandardAction(
```

Comment:
Technically, a reducer is a function that takes actions and mutates the store. Wouldn't a name like `itwLifecycleStoresReset` more appropriate?

Reply:
It makes sense, naming is not always easy! Addressed in 0fd3d7de2bb89ea340cf40539619b951fb8862b1.